assets/stylesheets/directly.scss

```diff
@@ -33,7 +33,7 @@
 // QUIT AND RESTART YOUR BROWSER NORMALLY WHEN YOU'RE DONE MAKING LOCAL CSS CHANGES.
 
 @import 'shared/mixins/mixins';
-@import 'shared/colors';
+@import '../../node_modules/@automattic/calypso-shared-style/src/shared/colors';
```

Comment:
:nauseated_face: Webpack alias?

Reply:
directly is built directly (duck) by node-sass, so webpack won't help

Reply:
might be able to use something like https://github.com/anarh/node-sass-import

Reply:
or https://www.npmjs.com/package/node-sass-package-importer if we like the `~`

Reply:
Using the `node-sass-package-importer` to enable tilde imports compatible with webpack's `css-loader` would be awesome.

TIL that `node-sass` supports [importer and function](https://github.com/sass/node-sass#importer--v200---experimental) plugins written in JS. Given that it's a wrapper around a binary compiled from C, I didn't expect such a nice extensibility.

Reply:
done in https://github.com/Automattic/wp-calypso/pull/30969

Reply:
Thanks, verified that https://github.com/Automattic/wp-calypso/pull/30933#issuecomment-466574258 works when this branch is rebased on #30969.